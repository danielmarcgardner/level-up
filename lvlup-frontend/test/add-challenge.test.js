import AddChallenge from '../src/components/Admin/admin-add-challenge/add-challenge-main';
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

test('AddChallenge should render', () => {
  const addchallenge = shallow(
    <AddChallenge />,
  );
  expect(toJson(addchallenge)).toMatchSnapshot();
});
