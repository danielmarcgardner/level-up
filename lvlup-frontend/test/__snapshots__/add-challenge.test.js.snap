exports[`test AddChallenge should render 1`] = `
<div
  className="add-challenge">
  <Connect(ReduxForm) />
</div>
`;
